addon/components/paper-item.js

```diff
@@ -82,10 +82,12 @@ export default Component.extend(RippleMixin, ProxyMixin, {
         let el = component.$();
         set(this, 'mouseActive', false);
         el.on('mousedown', ()=> {
-          set(this, 'mouseActive', true);
-          run.later(()=> {
-            set(this, 'mouseActive', false);
-          }, 100);
+          if (!this.get('isDestroyed')) {
```

Comment:
You can't just check this here. You need to check inside the run later or the error will still occur if the action is async which is most cases.